teku/src/main/java/tech/pegasys/teku/cli/options/BeaconRestApiOptions.java

```diff
@@ -51,6 +51,16 @@ public class BeaconRestApiOptions {
       arity = "0..1")
   private boolean restApiEnabled = false;
 
+  @Option(
+      names = {"--rest-api-light-client-enabled"},
+      paramLabel = "<BOOLEAN>",
+      showDefaultValue = Visibility.ALWAYS,
+      description =
+          "Enables Beacon Rest API light client endpoints and storage of light client data.",
+      fallbackValue = "true",
+      arity = "0..1")
```

Comment:
i think once we're all fully done, we'll enable this, but for now make it a -X option and mark hidden, it's basically a feature toggle...

Reply:
the variable name and config can stay the same, just make it `-Xrest-api-light-client-enabled`

Reply:
Thanks @rolfyone I updated the flag name